TIPS.md

```diff
@@ -0,0 +1,31 @@
+Instructions for play
+
+- While some parts of the mod are scripted, many parts (such as exploring and building/placing of invaders after setup) are not
+- When playing single player you can ignore the ready token and the "ready" text at the upper right (it's used for coordination among players)
+- Do not hit the rewind/fastforward (they do not work well with scripted mods)
+- To look at the reverse side of your spirit panel press Alt+Shift (default keybinding in TTS) and mouse over the spirit panel 
+- To zoom in on a card hold the Alt key (default keybinding in TTS) and mouse over the card/object
+
+Intermediate Tips:
+- Press numpad keys (as shown in the bottom right) to spawn new pieces
+  - Make sure numlock is enabled
+  - Note that these scripting keys can be rebound in the TTS settings
+- Lock pieces (right-click and select lock)
+- Mark damage on pieces by changing the state
+  - i.e. to mark one damage on a town, right click the town and change the state to 2
+  - You can also press a number key to change the state (so press 2 to change the state to 2)
+  - Note that invaders that have extra health or deal extra damage are rendered in a different color (i.e. red for england buildings, and blue for russian explorers)
+- Defend values can also be set as a state
+- To prevent a piece from going away during time passes you should lock it (right-click and select lock)
+- You can use energy tokens instead of the counter by deleting the energy counter (right-click on the energy counter and press delete)
+  
+Advanced Tips:
+- UI elements can be enabled and disabled with the little triangle near the top-right of your screen
+- You can play double-handed (i.e. play 2 or more spirits) by changing your TTS player color in the top-right of the screen and choosing another spirit
+  - Note that your UI elements are tracked per spirit
+- You can bind keys to many other actions with options->keys (TODO check name)
```

Comment:
Options -> Game Keys

Reply:
Perfect, thanks. Updated in 4abc225675c64008c6f7084e2cea49504dedb219